### DISCUSSION TITLE
Unable to Update Spending Limit

### DISCUSSION THREAD

**Discussion by danielmohemmed** [community/community#29803] [Actions]
Guys, I'm facing an issue while updating the Spending Limit for Action runners, as it says "Spending limit amount cannot exceed ....", is there any idea how I can fix it?

**Comment by Akash1134** [comment]
Hi @danielmohemmed ,

Could you share the exact error message you're getting while updating the [Spending Limits for Action Runners](https://docs.github.com/en/billing/managing-billing-for-github-actions/managing-your-spending-limit-for-github-actions#managing-the-spending-limit-for-github-actions-for-your-personal-account)? However, the best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)
